app.js

```diff
@@ -0,0 +1,91 @@
+const operations = require('./dboperations');
+const express = require('express');
+const cors = require('cors');
+const app = express();
+
+const appointment = require('./Appointment');
+const donor = require('./Donor');
+
+app.use(express.urlencoded());
+app.use(express.json());
+app.use(cors());
+
+
+
+const port = process.env.port || 3000;
+app.listen(port);
+console.log("Location API is running at port: " + port);
+
+app.use((request, response, next) =>{
+    console.log('middleware');
+    next();
+})
+
+
+//OPERATIONS
+
+//Location
+app.get('/locations', (request, response) => {
+    operations.getLocations().then(result => {
+        response.json(result[0]);
+    })
```

Comment:
Also don't forgot to do error handling as well. We can't assume that DB operations are 100% functional every time.